ada_feeding_perception/ada_feeding_perception/face_detection.py

```diff
@@ -403,9 +403,9 @@ def camera_callback(self, msg: CompressedImage):
                 (x, y, w, h) = faces[largest_face[1]]
                 cv2.rectangle(image_rgb, (x, y), (x + w, y + h), (255, 0, 0), 2)
 
-                # Find stomion (mouth center) in image
-                img_mouth_center = landmarks[largest_face[1]][0][66]
-                img_mouth_points = landmarks[largest_face[1]][0][48:68]
+                # Find marker between eyes. This is used to estimate stomion depth in the
+                # case that the stomion is hidden behind the fork.
```

Comment:
Nit: this is not done "in the case that the stomion is hidden behind the fork", it is done in all cases :) 

Perhaps reword to "This is used as a proxy to estimate stomion depth, because the stomion is often hidden behind the fork."